src/HashBus.Twitter.BackFill/App.cs

```diff
@@ -1,26 +1,30 @@
 ﻿namespace HashBus.Twitter.BackFill
 {
+    using System.Threading;
+    using System.Threading.Tasks;
     using HashBus.NServiceBusConfiguration;
     using HashBus.Twitter.Monitor.CatchUp.Commands;
     using NServiceBus;
     using NServiceBus.Persistence;
 
     class App
     {
-        public static void Run(
-            string nserviceBusConnectionString,
-            string endpointName,
-            string track,
-            long tweetId)
+        public static async Task Run(string nserviceBusConnectionString, string endpointName, string track, long tweetId, string monitorCatchUpAddress)
         {
-            var busConfiguration = new BusConfiguration();
-            busConfiguration.EndpointName(endpointName);
-            busConfiguration.UseSerialization<JsonSerializer>();
-            busConfiguration.EnableInstallers();
-            busConfiguration.UsePersistence<NHibernatePersistence>().ConnectionString(nserviceBusConnectionString);
-            busConfiguration.ApplyMessageConventions();
+            var endpointConfiguration = new EndpointConfiguration(endpointName);
+            endpointConfiguration.UseSerialization<JsonSerializer>();
+            endpointConfiguration.EnableInstallers();
+            endpointConfiguration.UsePersistence<NHibernatePersistence>().ConnectionString(nserviceBusConnectionString);
+            endpointConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyErrorAndAuditQueueSettings();
 
-            using (var bus = Bus.Create(busConfiguration).Start())
+            var transportExtensions = endpointConfiguration.UseTransport<MsmqTransport>();
+            var routing = transportExtensions.Routing();

```

Comment:
Please insert a blank line before this line (for consistency with other endpoints).